### ISSUE TITLE
Are you plan to release pretrained models in paper

### ISSUE THREAD

**Issue by doublexxking** [SimonGiebenhain/NPHM#10]
Great work!

As the title mentioned, the scores of pretrained models in paper are much better than github. 
are you plan to release the models that trained in all of identities?

Thanks.

**Comment by SimonGiebenhain** [comment]
Hi, 

The model is the same (just the model on GitHub is trained on the full training set, while the model in the paper was only trained on 87 identities which we had the time of submission).

Furthermore, we made a mistake and reported and the numbers in the paper are not computed in metric space.

Finally, we changed the observations by varying the camera position randomly, instead of always having a perfect frontal camera. In our opinion this makes the scenario more realistic.

I hope that helps.
